### ISSUE TITLE
Add MNIST Notebook

### ISSUE THREAD

**Issue by madhavajay** [OpenMined/PySyft#4660]
## Feature Description
Convert the existing Colab notebook into a proper git version controlled notebook.
Existing one here:
https://colab.research.google.com/drive/1SMNmxJk_qRTe15Rkd65s5Q7ZlipnUGCT

Split into both DO (Data Owner) and DS (Data Scientist) notebooks reflecting the two different notebooks that users will play with.

Test set to be on DS side and Training set to be on DO side.

## Definition of Done
The partially runnable MNIST notebook should be in the examples/pytorch/mnist folder and a README.md should be added in the parent examples/pytorch directory with a link to the original example and our notebook.

**Comment by madhavajay** [comment]
Test set to be on DS side and Training set to be on DO side.
